File: CPP/TeknisiIT.cpp
//Keahlian, SistemYgDikelola
#pragma once
#include <iostream>
#include <string>
#include "Tendik.cpp"

using namespace std;

class TeknisiIT : public Tendik{
    private:
        string Keahlian;
        string SistemYgDikelola;
    
    public:
        TeknisiIT(){
            setNIK("");
            setNama("");
            setJenisKelamin("");
            setNIP("");
            setJabatan("");
            Keahlian = "";
            SistemYgDikelola = "";
        };
        TeknisiIT(string nik, string nama, string jk, string nip, string jabatan, string keahlian, string sistem, UnitKerja unit){
            setNIK(nik);
            setNama(nama);
            setJenisKelamin(jk);
            setNIP(nip);
            setJabatan(jabatan);
            setUnit(unit);
            Keahlian = keahlian;
            SistemYgDikelola = sistem;
        }

        //setter
        void setKeahlian(string nik){
            Keahlian = nik;
        }
        void setSistemYgDikelola(string nik){
            SistemYgDikelola = nik;
        }

        //getter
        string getKeahlian(){
            return Keahlian;
        }
        string getSistemYgDikelola(){
            return SistemYgDikelola;
        }
        ~TeknisiIT(){
        }
};

File: JAVA/TeknisiIT.java
//Keahlian, SistemYgDikelola

public class TeknisiIT extends Tendik{
    private String Keahlian;
    private String SistemYgDikelola;

    public TeknisiIT(){
        setNIK("");
        setNama("");
        setJenisKelamin("");
        setNIP("");
        setJabatan("");
        this.Keahlian = "";
        this.SistemYgDikelola = "";
    }

    public TeknisiIT(String nik, String nama, String jk, String nip, String jabatan, String keahlian, String sistem, UnitKerja unit){
        setNIK(nik);
        setNama(nama);
        setJenisKelamin(jk);
        setNIP(nip);
        setJabatan(jabatan);
        setUnit(unit);
        this.Keahlian = keahlian;
        this.SistemYgDikelola = sistem;
    }

    //setter
    public void setKeahlian(String nik){
        this.Keahlian = nik;
    }

    public void setSistemYgDikelola(String nik){
        this.SistemYgDikelola = nik;
    }

    //getter
    public String getKeahlian(){
        return this.Keahlian;
    }

    public String getSistemYgDikelola(){
        return this.SistemYgDikelola;
    }
}